### PR TITLE
Handling user cancellation of Apple Pay #85

### PR DESCRIPTION
**Pull request by radhakrishnant2s**
1. When ever the user manually dismisses apple pay, the event should be passed to the caller via callback
2. This is impacting the React native library

### CODE REVIEW

File: JudoKitObjC.podspec [hunk 1]
```diff
@@ -1,11 +1,11 @@
 Pod::Spec.new do |s|
   s.name                  = 'JudoKitObjC'
-  s.version               = '10.0.2'
+  s.version               = '10.0'
```
Comment (mpetrenco) [inline JudoKitObjC.podspec:3]: Can we revert this?

File: JudoKitObjC.podspec [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
   s.summary               = 'Judo Pay Full iOS Client Kit'
   s.homepage              = 'https://www.judopay.com/'
   s.license               = 'MIT'
   s.author                = { "Judopay" => 'developersupport@judopayments.com' }
-  s.source                = { :git => 'https://github.com/Judopay/JudoKitObjC.git', :tag => s.version.to_s }
+  s.source                = { :git => 'https://github.com/radhakrishnant2s/JudoKitObjC.git', :tag => s.version.to_s }
```
Comment (mpetrenco) [inline JudoKitObjC.podspec:8]: Can we revert this as well?

**Comment by mpetrenco** [comment]
@radhakrishnant2s please note that this is an outdated iOS library, used mainly to support older merchants that are yet to transition to the new SDK.

Our React Native SDK uses the new [JudoKit-iOS](https://github.com/Judopay/JudoKit-iOS) library.

File: JudoKitObjC.podspec (hunk 2 above)
Comment (radhakrishnant2s) [reply JudoKitObjC.podspec:8]: closing this PR as I am going to adopt the latest react native

**Pull request closed without merging**